cpu/atmega_common/cpu.c

```diff
@@ -110,28 +120,30 @@ void cpu_init(void)
  * EIFR – External Interrupt Flag Register
  * PCIFR – Pin Change Interrupt Flag Register
  */
-ISR(BADISR_vect){
-
+ISR(BADISR_vect)
+{
     _reset_cause();
 
     printf_P(PSTR("FATAL ERROR: BADISR_vect called, unprocessed Interrupt.\n"
                   "STOP Execution.\n"));
```

Comment:
This error message could probably be a ```core_panic``` with reason ```PANIC_DUMMY_HANDLER```.

It would need to go where the ```while (1) {}``` is, because a panic halts the system.

Reply:
Ok, I'll modify that part and make the corresponding tests.

Reply:
I was thinking, is there an easy way to trigger this? So we can make a proper test.